info.py

```diff
@@ -627,6 +636,58 @@ def plot(tl, device_gdf, device_to_lines, extent=[-125, -66.5, 24, 50]):
       # Optional plot the data for TVA and NERC on the same plot
       plot(tl_gdf_subset, device_gdf, device_to_lines, extent=[-125, -67, 25.5, 49.5])
 
+# Code for std and peak of GIC
+# TODO: make less messy, right now depends on main.py running first to make all.pkl, and main.py needs info.py to run first
+def add_gic_parameters(info_df, all_file):
```

Comment:
std and min/max computed in stats:

https://github.com/lucywilkerson/SWERVE/blob/generalize/swerve/site_stats.py#L60

could add |max| entry. We can discuss how do deal with this today.